analyzers/src/SonarAnalyzer.Common/Rules/Utilities/MetricsAnalyzerBase.cs

```diff
@@ -40,13 +40,10 @@ public abstract class MetricsAnalyzerBase : UtilityAnalyzerBase<MetricsInfo>
 
         protected abstract MetricsBase GetMetrics(SyntaxTree syntaxTree, SemanticModel semanticModel);
 
-        protected sealed override MetricsInfo GetMessage(SyntaxTree syntaxTree, SemanticModel semanticModel)
+        protected sealed override MetricsInfo CreateMessage(SyntaxTree syntaxTree, SemanticModel semanticModel)
```

Comment:
clean as you code: should add a test for this, you can create a small syntax tree and check the results, it shouldn't take long...